### PR TITLE
[Serializer] minor: Remove an extra space in a YamlFileLoader error

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Branch? | 3.1 |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

**Comment by javiereguiluz** [comment]
:+1:

**Comment by xabbuh** [comment]
👍

File: src/Symfony/Component/Serializer/Mapping/Loader/YamlFileLoader.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
 
                     if (isset($data['max_depth'])) {
                         if (!is_int($data['max_depth'])) {
-                            throw new MappingException('The "max_depth" value must an integer  in "%s" for the attribute "%s" of the class "%s".', $this->file, $attribute, $classMetadata->getName());
+                            throw new MappingException('The "max_depth" value must an integer in "%s" for the attribute "%s" of the class "%s".', $this->file, $attribute, $classMetadata->getName());
```
Comment (fabpot) [inline src/Symfony/Component/Serializer/Mapping/Loader/YamlFileLoader.php]: must be?

**Comment by fabpot** [comment]
Thank you @dunglas.

**Pull request closed without merging**